### DISCUSSION TITLE
Trigger LiveTile code from App.xaml.cs (OnLaunched) in the UWP Project.

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#8829] [Q&A] [answered]
Dear all, maybe stupid question. But i want to put the livetile code in the uwp project so the rest wont complain.. but my app.xaml.cs is in the shared and now: how can i trigger the code in the uwp project from OnLaunched in shared App.xaml.cs?

`... OnLaunced()
{
    #if(windows_UWP)
           Singleton<LiveTileService>.Instance.SampleUpdate(); //code is in uwp proj.
    #endif
}`

**Comment by Daantje007** [comment, marked as answer]
aah i found you need to add the refferences platform specific code like..

#if(windows_UWP)
using Namespace.folder ()
#endif

my bad, good luck

**Reply by jeromelaban** [reply]
Indeed, that is a generally good way for solving this. See https://platform.uno/docs/articles/platform-specific-csharp.html for more details.
